interfaces/python/helicsPython.i

```diff
@@ -3,7 +3,7 @@
 %include "cstring.i"
 %include "python_maps.i"
 
-%include "../helics.i"
+%include "helics.i"
```

Comment:
Did we move the file or is that now on the include path for swig?

Reply:
It's on the include path for swig.